src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -334,6 +335,12 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Throttling.TimeIntervalInMs", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public TimeSetting? ThrottlingTimeInterval { get; protected set; }
 
+        [Description("Index engine for AutoIndexes")]
```

Comment:
For new auto indexes. I believe this should control the search engine only at the creation time of an index. Once an index is created then we must not change its search engine.